<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTransferDialog</class>
 <widget class="QDialog" name="FileTransferDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Transfer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_task">
        <property name="text">
         <string>Current Task: Creating a list of files to copy...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_source">
        <property name="text">
         <string>From: ...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_target">
        <property name="text">
         <string>To: ...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_speed">
        <property name="text">
         <string>Speed: ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_progress_total">
          <property name="text">
           <string>Total:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_progress_current">
          <property name="text">
           <string>Item:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QProgressBar" name="progress_total">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progress_current">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
